### ISSUE TITLE
Cascade compile command failed

### ISSUE THREAD

**Issue by olafurpg** [scalameta/metals#3193]
**Describe the bug**

Triggering the Cascade Compile command reports an error that the command failed.

**To Reproduce**

I haven't found a reliable reproduction yet. The modal appeared when I triggered the command, but the error doesn't happen every time I trigger the command.

**Expected behavior**

The error message is not actionable so I don't expect the modal to show up. Modal prompts should be used very sparingly for highly actionable errors. It would be better if the message got logged instead.

**Screenshots**

<img width="896" alt="image" src="https://user-images.githubusercontent.com/1408093/136703209-3c1a77a5-a497-4e27-8f0c-246294f22f86.png">

**Installation:**

- Operating system: macOS
- Editor: Visual Studio Code
- Metals version: v0.10.7

**Additional context**

I'm using sbt server and without SemanticDB enabled.

**Search terms**

**Comment by tgodzik** [comment]
Thanks for finding this! Wow, i haven't seen issues like this aside from when the Metals extension was not loaded. This is on the VS Code side, so there could be some logs in the metals log tab or extension host one.

Could you check it next time it happens? It should be reproducible, I will take a look next week if no one manages to check out before.

**Comment by olafurpg** [comment]
@dos65 I don't think that stack trace is related to the cascade compile failure, but maybe it is 🤷

**Comment by olafurpg** [comment]
I'm seeing the same error with other commands like "Connect to build server" 

<img width="896" alt="image" src="https://user-images.githubusercontent.com/1408093/137623796-3a337d84-5429-498f-967c-1c39d2e51466.png">

We can transfer this issue to the VS Code extension since there's nothing wrong about the Metals side (I think)
